src/App.tsx

```diff
@@ -2,26 +2,44 @@ import { Map } from "components/Map";
 import { getSpots, Spot } from "datasource";
 import "leaflet/dist/leaflet.css";
 import React, { useEffect, useState } from "react";
+import { act } from "react-dom/test-utils";
 import { Marker } from "react-leaflet";
+import MarkerClusterGroup from "react-leaflet-markercluster";
 
 const App = () => {
   const [spots, setSpots] = useState<Spot[]>([]);
 
   useEffect(() => {
     getSpots().then(spots => {
-      setSpots(spots);
+      act(() => {
```

Comment:
`act` and other methods from `test-utils` shouldn't be used outside of tests. It has no effect in here.